### PR TITLE
Bugfix/scope edge workarounds

### LINKED ISSUE #309: Edge-16 WTF menu bug
#304  isn't completely fixed.

If the user clicks a left-side menu item, it's not shown, if he clicks a right-side menu item it is shown 😱 

edge-16-menu-wtf-small

### LINKED ISSUE #367: EDGE 16 sub navigation not visible upon first click
## Are you reporting a bug or a feature?

 - [x] Bug

## Expected Behavior

The sub navigation should be properly toggling, also upon first click in every browser.

## Actual Behavior

The sub navigation isn't visible upon first click on **EDGE 16**.

## Steps to Reproduce the Problem

1. Go to https://www.acc.axa.ch

ezgif com-resize

## Specifications

 - Browser: EDGE
 - Browser Version: 16

### PR DESCRIPTION
**Pull request by AndyOGo**
Fixes #309 , fixes #367 

Changes proposed in this pull request:

 - only execute workaround if the browser is actually **EDGE**

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
 
 # How Has This Been Tested?
 
not yet.
 
 # Checklist:
 
all done

### CODE REVIEW

File: src/components/m-header-navigation/js/header-navigation.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { add, remove } from '../../../js/class-list';
 import { requestAnimationFrame } from '../../../js/request-animation-frame';
 import forceRepaint from '../../../js/force-repaint';
 import UiEvents from '../../../js/ui-events';
+import isEdge from '../../../js/is-edge';
```
Comment (LucaMele) [inline src/components/m-header-navigation/js/header-navigation.js]: please add it in the shame.js

Comment (AndyOGo) [reply src/components/m-header-navigation/js/header-navigation.js]: hmh, let's rename the file to `is-edge-SHAME.js`?

Comment (AndyOGo) [reply src/components/m-header-navigation/js/header-navigation.js]: or we could also introduce a folder called `shame/` to group all shame code

Comment (LucaMele) [reply src/components/m-header-navigation/js/header-navigation.js]: i would put all the shame stuff in one file, very dirty so that it looks bad

Comment (LucaMele) [reply src/components/m-header-navigation/js/header-navigation.js]: i hope that we remove that code soon

Comment (LucaMele) [reply src/components/m-header-navigation/js/header-navigation.js]: thats ok with the folder as well

Comment (AndyOGo) [reply src/components/m-header-navigation/js/header-navigation.js]: Okidok, I will move it into a separate folder.

Comment (LucaMele) [reply src/components/m-header-navigation/js/header-navigation.js]: thanks :)

Comment (AndyOGo) [reply src/components/m-header-navigation/js/header-navigation.js]: Done :)

**Comment by AndyOGo** [comment]
@LucaMele 
Before we merge this, we should make sure, that only Edge has those menu bugs.

**Comment by LucaMele** [comment]
ok then we have to test it on BS

**Comment by LucaMele** [comment]
bildschirmfoto 2018-03-29 um 10 10 02

**Comment by AndyOGo** [comment]
any updates on this?

**Pull request merged**